docs/introduction/about-plugins.md

```diff
@@ -30,7 +30,7 @@ For example, the code snippet above is equivalent to:
 
 ```js
 plugins: [
-    [ /mypath\/**.css/, CSSPlugin() ]
+    [ /mypath\/\*\*\.css/, CSSPlugin() ]
```

Comment:
it's a valid RegExp I believe we should have `.*\.css`

Reply:
Nope, the equivalent Regexp of `"mypath/**.css"` is ` /mypath\/\*\*\.css/`.

Example (see also https://www.google.com/search?q=js+escape+string+to+regex and https://stackoverflow.com/a/6969486):

    $ node
    > function escapeRegExp(str) {
    ...   return str.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, "\\$&");
    ... }
    undefined
    > new RegExp(escapeRegExp("mypath/**.css"))
    /mypath\/\*\*\.css/

Reply:
The characters '/' and '*' and '.' all need to be escaped.

Reply:
I agree w/ @felixrabe .

Reply:
But ` /mypath\/\*\*\.css/` isn't a string. It's a REGEXP